src/components/intro.docs.stories.mdx

```diff
@@ -10,6 +10,7 @@ https://github.com/cbinsights/form-design-system
 
 - [📖 Dictionary](https://cbinsights.github.io/form-design-system/fds-dictionary)
 - [🎨 Base Styles](https://cbinsights.github.io/form-design-system/fds-styles)
+- [⚛️ React Components](#components-forms)
```

Comment:
This link doesn't actually work right now. I need to fix this (it's simple, I just don't want to make any more commits on this for now)

Reply:
separate PR works for me